CMakeLists.txt

```diff
@@ -2,21 +2,32 @@
 cmake_minimum_required(VERSION 3.18)
 set_property(GLOBAL PROPERTY USE_FOLDERS ON)
 
-# CMakeExtensions
 include(FetchContent)
+
+# Modules declaration
 FetchContent_Declare(cmake-extensions
     GIT_REPOSITORY https://github.com/BabylonJS/CMakeExtensions.git
     GIT_TAG 7bdd0664181ce3d90dc94474464cea9b25b22db7)
-message(STATUS "Fetching cmake-extensions")
-FetchContent_MakeAvailable(cmake-extensions)
-message(STATUS "Fetching cmake-extensions - done")
-
-# JSRuntime Host
+FetchContent_Declare(ios-cmake
+    GIT_REPOSITORY https://github.com/leetal/ios-cmake.git
+    GIT_TAG 04d91f6675dabb3c97df346a32f6184b0a7ef845)
 FetchContent_Declare(jsruntimehost
     GIT_REPOSITORY https://github.com/BabylonJS/JsRuntimeHost.git
-    GIT_TAG 7d4c1966b54dca290be44646998cbeec431fca50)
+    GIT_TAG d1667b827c5b80c07c959780e582ea0d9a11c0ad)
+FetchContent_Declare(AndroidExtensions
+    GIT_REPOSITORY https://github.com/BabylonJS/AndroidExtensions.git
+    GIT_TAG 883c4a286116e51ee989e39ee8e216d632997329)
 set(JSRUNTIMEHOST_TESTS OFF)
-FetchContent_MakeAvailable(jsruntimehost)
+
+set(CONTENT_TO_FETCH cmake-extensions jsruntimehost)
+
+if(IOS)
+    FetchContent_MakeAvailable(ios-cmake)
+    set(CMAKE_TOOLCHAIN_FILE "${ios-cmake_SOURCE_DIR}/ios.toolchain.cmake" CACHE PATH "")
+    set(PLATFORM "OS64COMBINED" CACHE STRING "")
+    set(DEPLOYMENT_TARGET "12" CACHE STRING "")
+    set(ENABLE_ARC OFF CACHE STRING "Enables or disables ARC support.")
```

Comment:
Why turn this off by default? If bgfx needs this to be off, I think we should only set it off for bgfx.

Reply:
I tried without success. if I force the flag for bgfx, then the brige is failing to compile. in the doc, the command line disables arc : https://github.com/BabylonJS/BabylonNative/blob/master/BUILDING.md#building-on-macos-targeting-ios